frontend/src/scenes/data-management/definition/DefinitionView.tsx

```diff
@@ -133,20 +132,6 @@ export function DefinitionView(props: DefinitionLogicProps = {}): JSX.Element {
             />
 
             <div className="space-y-2">
-                <EditableField
```

Comment:
Will this mean that the description won't show anymore? 

![Screenshot 2024-06-04 at 2 41 07 PM](https://github.com/PostHog/posthog/assets/6392501/cee0ad95-da30-4906-a06a-5617c9a99e96)

![Screenshot 2024-06-04 at 2 41 13 PM](https://github.com/PostHog/posthog/assets/6392501/281936b7-4643-4a3b-94b9-fba33915392d)

Reply:
How were you able to edit that?? I couldn't edit it haha

Reply:
"Edit" in the top right, and then a textarea appears. I can't edit the field inline.

Reply:
OH. You have to have the INGESTION_TAXONOMY feature to see the edit button. 

No edit button without INGESTION_TAXONOMY:

![Arc 2024-06-04 11 47 41](https://github.com/PostHog/posthog/assets/18598166/024dc24f-027c-4193-ac0a-1ddb8df8968a)

Will update, thanks!